src/miner.app.src

```diff
@@ -36,7 +36,17 @@
     grpc_client
    ]},
   {included_applications, [blockchain, sibyl]},
-  {env,[]},
+  {env,
```

Comment:
add a comment here that says it needs to be updated in the configs as well? and a comment in the configs to point here?